great_expectations/expectations/core/unexpected_rows_expectation.py

```diff
@@ -122,15 +121,8 @@ def _prescriptive_template(
         cls,
         renderer_configuration: RendererConfiguration,
     ) -> RendererConfiguration:
-        add_param_args: AddParamArgs = (
-            ("description", RendererValueType.STRING),
-            ("unexpected_rows_query", RendererValueType.STRING),
-        )
-        for name, param_type in add_param_args:
-            renderer_configuration.add_param(name=name, param_type=param_type)
-
-        renderer_configuration.template_str = (
-            "$description" if renderer_configuration.params.description else ""
```

Comment:
The individual renderers no longer need to add this logic. It is handled by `RendererConfiguration` automatically.